torchvision/io/video.py

```diff
@@ -232,6 +248,8 @@ def read_video_timestamps(filename):
     -------
     pts : List[int]
         presentation timestamps for each one of the frames in the video.
+        Note that these are returned as the TRUE timestamps, i.e.
```

Comment:
can you mention `the true timestamps in seconds`?

Reply:
yup.